### PR TITLE
add support for sending to multiple recipients via sendmail

### PR DESCRIPTION
**Pull request by monperrus**
if the `to` config contains multiple emails such as `x@foo.com,y@bar.com`

### CODE REVIEW

**Comment by Jamstah** [comment]
Does this also work for the smtp method? That field should probably work the same for both mechanisms.

This PR is missing documentation, and an entry in the changelog.

**Comment by monperrus** [comment]
> Does this also work for the smtp method?

Yes, it already works for SMTP

> This PR is missing documentation and an entry in the changelog.

Done.

Thanks!

File: docs/source/reporters.rst [hunk 1]
```diff
@@ -69,6 +69,7 @@ At the moment, the following reporters are built-in:
 - **prowl**: Send a detailed notification via prowlapp.com
 - **pushbullet**: Send summary via pushbullet.com
 - **pushover**: Send summary via pushover.net
+- **sendmail**: Pipe a message to the system sendmail command
```
Comment (Jamstah) [inline docs/source/reporters.rst:72]: There isn't a reporter called `sendmail`, its a method of the `email` reporter.

I think we need a new section like [this one](https://urlwatch.readthedocs.io/en/latest/reporters.html#e-mail-via-gmail-smtp), but titled "E-Mail via sendmail" or something.

**Review by Jamstah — CHANGES_REQUESTED** [review]

File: docs/source/reporters.rst (hunk 1 above)
Comment (monperrus) [reply docs/source/reporters.rst]: @Jamstah done!

File: docs/source/reporters.rst [hunk 2]
```diff
@@ -290,6 +290,20 @@ public Matrix room, as the messages quickly become noisy:
...
+---------------------
+
+You can send email via the system's sendmail command provided by the MTA. You need to set "method: sendmail" in the config file.
+
+.. code:: yaml
+
+    report:
+      email:
+        enabled: true
+        from: 'postmaster@example.com'
```
Comment (Jamstah) [inline docs/source/reporters.rst:303]: Should probably have a `to:` in this example as well.

Comment (monperrus) [reply docs/source/reporters.rst:303]: done

File: CHANGELOG.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ The format mostly follows [Keep a Changelog](http://keepachangelog.com/en/1.0.0/
 
 ### Fixed
 
+- Fix multiple recipients for sendmail reporter (#797)
```
Comment (thp) [inline CHANGELOG.md:21]:
```suggestion
- `email` reporter: Allow multiple recipients for `sendmail` method (#797, by monperrus)
```

File: docs/source/reporters.rst [hunk 4]
```diff
@@ -290,6 +290,21 @@ public Matrix room, as the messages quickly become noisy:
      minimal: true
      enabled: true
 
+E-Mail via sendmail
+---------------------
+
+You can send email via the system's sendmail command provided by the MTA. You need to set "method: sendmail" in the config file.
```
Comment (thp) [inline docs/source/reporters.rst:296]: Formatting nitpicks:
 
```suggestion
You can send email via the system's ``sendmail`` command provided by the MTA. You need to set ``method: sendmail`` in the config file:
```

**Review by thp — CHANGES_REQUESTED** [review]
Thanks, two small comments.

**Comment by monperrus** [comment]
Thanks for the review, applied the changes.

**Review by thp — APPROVED** [review]
Thanks!

**Comment by monperrus** [comment]
thank you!

**Pull request merged**